### PR TITLE
Fix mosh install and use newest version of dokku.

### PR DESCRIPTION
**Pull request by benjroy**
I was able to get Riker to work for a few deploys after some tinkering.

apt-get install python-software-properties was failing when I first tried it.  Apt needed an update first for it to install properly.

Also, I was getting nginx errors on my deployed instances until I updated the version of dokku installed on those instances.

These two changes allowed everything to complete successfully.

### CODE REVIEW

File: riker/api.py [hunk 1]
```diff
@@ -489,6 +489,7 @@ def install_dokku(self):
     # @synchronize('install_mosh.lock', is_remote=True)
     def install_mosh(self):
         log('info', 'Installing mosh', show_header=True)
+        sudo('apt-get update -y')
         sudo('apt-get install -y python-software-properties')
         sudo('add-apt-repository -y ppa:keithw/mosh')
         sudo('apt-get update -y')
```
Comment (benjroy) [inline riker/api.py]: It seems redundant to have this second update here.  I just slammed another apt-get update in at the start.
Maybe a better way would be to:
```
         log('info', 'Installing mosh', show_header=True)
         sudo('add-apt-repository -y ppa:keithw/mosh')
         sudo('apt-get update -y')
         sudo('apt-get install -y python-software-properties')
         sudo('apt-get install -y mosh')
```

That would work, yea?

Comment (jschementi) [inline riker/api.py]: That updated ordering looks good, I'd try that.

File: riker/api.py [hunk 2]
```diff
@@ -30,7 +30,7 @@
 from retry import synchronize
 
 import fabric
-fabric.state.output.everything = False
+fabric.state.output.everything = True
```
Comment (jschementi) [inline riker/api.py]: Just like the verbose output?

**Comment by jschementi** [comment]
Looks great, thanks for the pull request!

File: riker/api.py (hunk 2 above)
Comment (benjroy) [inline riker/api.py]: I'm not the biggest fan of OUTPUT-AS-VERBOSE-AS-YOU-POSSIBLY-CAN, but with this set to false, then any time i ran a successful ```riker dokku "<cmd>" --app <app> --env <env>```, I got no output in my terminal's stdout.  fabric was sucking up everything.

Flipping the logging here to True is a dumb hammer.  it let's me use the ```riker dokku``` command, and it is also extremely helpful to see the buildstep output building my npm applications during the deploy process.

This ```fabric.state.output.everything = True``` hammer is a little too verbose, and I am unfortunately not all that familiar with fabric.  Are you aware of any different settings that would be more useful and keep the output pretty?

File: riker/api.py [hunk 3]
```diff
@@ -16,7 +16,7 @@
 from boto.ec2.autoscale import LaunchConfiguration
 from boto.ec2.autoscale import AutoScalingGroup
 from boto.ec2.elb.attributes import ConnectionDrainingAttribute
-from fabric.api import task, run, local, env, sudo, lcd, execute, put
+from fabric.api import task, run, local, env, sudo, lcd, execute, put, settings
```
Comment (benjroy) [inline riker/api.py]: I need to remove the changes on this line.  I didn't end up using the settings from fabric.api.

File: riker/api.py (hunk 2 above)
Comment (jschementi) [inline riker/api.py]: Ah, I see. OK, that'll have to do for now until we can get `riker dokku` to work without it.

**Comment by jschementi** [comment]
Tested, works great!

**Pull request merged**